docs/export/README.md

```diff
@@ -0,0 +1,100 @@
+<!--
+title: Export metrics
+description: "Archive your Netdata metrics to multiple external time series databases for long-term storage or further analysis."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/export/README.md
+-->
+
+# Export metrics
+
+One of Netdata's pillars is interoperability with other monitoring and visualization solutions. To this end, you can use
+the Agent's [exporting engine](/exporting/README.md) to send metrics to multiple external databases/services in
+parallel. Once you connect Netdata metrics to other solutions, you can apply machine learning analysis or correlation
+with other tools, such as application tracing.
+
+The exporting engine supports a number of connectors, including AWS Kinesis Data Streams, Graphite, JSON, MongoDB,
+OpenTSDB, Prometheus remote write, and more, via exporting **connectors**. These connectors help you seamlessly send
+Netdata metrics to more than 20 different endpoints, including every [service that
+supports](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage) Prometheus remote write. See
+the [exporting reference guide](/exporting/README.md) for the full list.
+
+## Exporting quickstart
+
+Let's cover the process of enabling an exporting connector, using the Graphite connector as an example. These steps can
+be applied to other connectors as well.
+
+> If you are migrating from the deprecated backends system, this quickstart will also help you update your configuration
+> to the new format.
+
+Open the `exporting.conf` file with `edit-config`.
+
+```bash
+cd /etc/netdata   # Replace this path with your Netdata config directory, if different as found in the steps above
```

Comment:
"as found in the steps above" - which steps? It looks like this part should be removed.

Reply:
Good catch.